bucket/syncplay.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.6.9",
+    "description": "Solution to synchronize video playback across multiple instances of mpv, VLC, MPC-HC, MPC-BE and mplayer2 over the Internet.",
+    "homepage": "https://syncplay.pl/",
+    "license": "Apache-2.0",
+    "url": "https://github.com/Syncplay/syncplay/releases/download/v1.6.9/Syncplay-Portable-Client-1.6.9.zip",
```

Comment:
Why `portable`?

What does it mean? What needs to be persisted?

Reply:
Their page says the non-portable version stores connection settings in appdata

Reply:
Added persist for the client settings